runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkBatchPortablePipelineTranslator.java

```diff
@@ -461,7 +461,7 @@ private static void pruneOutput(
   }
 
   // Creates a mapping from local names to output tag integers.
-  private static BiMap<String, Integer> createOutputMap(Iterable<String> localOutputs) {
+  static BiMap<String, Integer> createOutputMap(Iterable<String> localOutputs) {
```

Comment:
This method is trivial enough that it's probably better to copy entirely instead of opening up to package wide access.

Reply:
I can make a copy, but IMO we should try to reduce code duplication between the translations.